.stylelintrc.json

```diff
@@ -11,7 +10,11 @@
     ],
     "rules": {
       "at-rule-no-unknown": null,
-      "scss/at-rule-no-unknown": true
+      "scss/at-rule-no-unknown": true,
+      "max-nesting-depth": 5,
+      "resolveNestedSelectors": true,
+      "scss/dollar-variable-pattern": "^foo",
```

Comment:
what does it mean?

Why did you add these rules?

Reply:
These were old settings that I didn't remove before commit. I restore to the previous state.